### PR TITLE
add errors collection for express server

### PR DESCRIPTION
**Pull request by hustKiwi**
ref: https://trello.com/c/MseXekg7/48-21-collecting-the-online-errors-by-using-sentry-2pt

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -16,6 +17,10 @@ const cfg = require('./config');
 const routes = require('./routes');
 
 
+Raven
+  .config('https://948bf30d5848446e9be83bdc8483a700:c925b1e6a4c8415d986a49df3195d788@sentry.io/220530')
```
Comment (neoxu314) [inline server/server.js]: should define a constant in config file

File: server/server.js [hunk 2]
```diff
@@ -36,12 +41,15 @@ class Server {
...
     app.set('view engine', 'ejs');
 
+    app.use(Raven.requestHandler());
     // compresses the content in gzip
     app.use(compression());
     app.use(bodyPaser());
     app.use('/static', express.static(`${__dirname}/../dist`));
     app.use(routes);
 
+    app.use(Raven.errorHandler());
```
Comment (neoxu314) [inline server/server.js]: should only collect errors in production enviroment.

File: server/server.js (hunk 1 above)
Comment (hustKiwi) [reply server/server.js]: fixed

File: server/server.js (hunk 2 above)
Comment (hustKiwi) [reply server/server.js]: fixed

**Review by neoxu314 — APPROVED** [review]

**Pull request merged**